recipes/list_items/contact_info/contact_info.mdx

```diff
@@ -20,7 +20,8 @@ The **right** slot works the same way, but its contents are placed to the right
 
 The **bottom** slot can be used to display content below the subtitle slot.
 
-Avatar is optional. When `avatarSrc` is specified, the avatar image will be displayed at the left.
+Avatar is optional. When `avatarSrc` is specified, the avatar image will be displayed at the left and
+is required to provide a value in `avatarInitials` to use in the alt attribute of the avatar image.
```

Comment:
"it is required"

Reply:
updated https://github.com/dialpad/dialtone-vue/pull/566/commits/9490e8f595a7c9b7c2b7dc0b4b1a937e1dcc9586